example/package.json

```
@@ -8,7 +8,7 @@
   "dependencies": {
     "react": "16.9.0",
     "react-native": "0.61.5",
-    "react-native-card-connect": "git+https://github.com/BigRoomStudios/react-native-card-connect.git#229fd6bce41ba3cbb86ef0197a799eb597e6b5ac"
+    "react-native-card-connect": "git+https://github.com/BigRoomStudios/react-native-card-connect.git#ff462b6e2fa79f47f0551b25a1d05cabed19b037"
```

```suggestion
    "react-native-card-connect": "git+https://github.com/BigRoomStudios/react-native-card-connect.git"
```
Commit before merging